app/helpers/save_this_page_helper.rb

```diff
@@ -15,6 +15,10 @@ def heading_text(options)
     options[:page_is_saved] ? I18n.t("components.save_this_page.page_was_saved_heading") : I18n.t("components.save_this_page.page_not_saved_heading")
   end
 
+  def saved_pages_link
+    "#{Plek.new.website_root}/account/saved-pages/"
```

Comment:
Nitpick: drop the trailing slash.  We don't usually display them for GOV.UK URLs.

Reply:
You could also inline this into `additional_text`

Reply:
Thanks, good shout, have done both